Fingerprint Vibration On Authentication

Exhigh opened the pull request:
Add Settings For Vibrate On Successful Fingerprint Authentication.

Exhigh committed:
Vibrate On Successful Fingerprint Auth

Add Settings For Vibrate On Successful Fingerprint Authentication.

Exhigh committed:
Update rr_lock.xml

JoshuaDoes commented:
I'm assuming you accidentally deleted your forked repo directly after creating the original pull request :P
I'll give this a test later today and if all goes well I'll merge

Exhigh commented:
Great . Thanks

Yeah That Happened . xD

Exhigh committed:
Update to Code

Exhigh committed:
Fingerprint Authentication Vibration #2

Exhigh committed:
Adding Controller to check fingerprintp sensor

To check fingerprint state

Exhigh committed:
Fingerprint Authentication Vibration

Made some additions to the file to implement logic on fingerprint authentication vibration.

Exhigh committed:
Made Small Changes

Made it so that the service is not active all the time

Exhigh committed:
Added Strings For Fingerprint 

Forgot to add them xD in the first place

Exhigh committed:
Made small Changes 

Called proper name of the initialized variable

Exhigh committed:
Enabling Freeform 

Add values to enable freeform in developer options

Exhigh committed:
Merge pull request #1 from Exhigh/patch-2

Fingerprint Authentication Vibration #2

Exhigh committed:
Merge pull request #2 from Exhigh/patch-3

Exhigh committed:
Merge pull request #3 from Exhigh/patch-4

Enabling Freeform
